artifact-manifest/net-monitor-image-nydus-ondemand-loading.json

```diff
@@ -0,0 +1,30 @@
+{
+  "schemaVersion": 3,
```

Comment:
I not sure of the reason we need to use a new version here.. would like to discuss this in detail..

Reply:
Someone provided feedback where this is a new schema, different from `oci-image` and suggested we should use version 3. Technically, it's the first `schemaVersion` of the artifact-manifest, so we just need to decide which to use.